bridge/engine/ActiveMediaList.cpp

```diff
@@ -217,63 +220,64 @@ bool ActiveMediaList::removeVideoParticipant(const size_t endpointIdHash)
 
 // note that  zero level is mainly produced by muted participants. All unmuted produce non zero level.
 // nonZerolevelWindow thus means how long a participant has been unmuted.
-void ActiveMediaList::updateLevels(const uint64_t timestampMs)
+void ActiveMediaList::updateLevels(const uint64_t timestamp)
 {
     for (auto& participantLevelEntry : _audioParticipants)
     {
         auto& participantLevels = participantLevelEntry.second;
         // Decay old max level over time (assuming process function called on average every 10ms)
-        // participantLevels._maxRecentLevel *= AudioParticipant::decayOfMaxLevel;
+        // participantLevels.maxRecentLevel *= AudioParticipant::MAX_LEVEL_DECAY;
         float averageLevelLongWindow =
-            static_cast<float>(participantLevels._totalLevelLongWindow) / participantLevels._levels.size();
-        participantLevels._maxRecentLevel -=
-            (participantLevels._maxRecentLevel - averageLevelLongWindow) * AudioParticipant::decayOfMaxLevel;
+            static_cast<float>(participantLevels.totalLevelLongWindow) / participantLevels.levels.size();
+        participantLevels.maxRecentLevel -=
+            (participantLevels.maxRecentLevel - averageLevelLongWindow) * AudioParticipant::MAX_LEVEL_DECAY;
         // Move old min level over time towards mean about 3dB per 3 seconds
-        participantLevels._noiseLevel = participantLevels._noiseLevel + AudioParticipant::noiseLevelRampup;
-        participantLevels._noiseLevel =
-            std::max(participantLevels._noiseLevel, static_cast<float>(AudioParticipant::minNoiseLevel));
+        participantLevels.noiseLevel = participantLevels.noiseLevel + AudioParticipant::NOISE_RAMPUP;
+        participantLevels.noiseLevel =
+            std::max(participantLevels.noiseLevel, static_cast<float>(AudioParticipant::MIN_NOISE));
     }
 
     for (AudioLevelEntry levelEntry; _incomingAudioLevels.pop(levelEntry);)
     {
-        auto participantLevelItr = _audioParticipants.find(levelEntry._participant);
+        auto participantLevelItr = _audioParticipants.find(levelEntry.participant);
         if (participantLevelItr == _audioParticipants.end())
         {
             continue;
         }
 
         auto& participantLevels = participantLevelItr->second;
+        participantLevels.lastPacketReceived = timestamp;
```

Comment:
This is reason I wanted to change to abs Timestamp. The Ms is a slight risk too as it will wrap differently from absTimestamp. In case the absTimeStamp starts at unfortunate value.

Reply:
This is reason I wanted to change to abs Timestamp. The Ms is a slight risk too as it will wrap differently from absTimestamp. In case the absTimeStamp starts at unfortunate value.